src/tests/multihost/alltests/test_ns_account_lock.py

```diff
@@ -22,6 +23,21 @@ def execute_cmd(multihost, command):
     return cmd
 
 
+def login_success(multihost, username, password):
+    """
+    Check if user can log in successfully
+.   """
+    client_hostip = multihost.client[0].ip
+    client = pexpect_ssh(client_hostip, username, password, debug=True)
+    try:
+        client.login(login_timeout=30, sync_multiplier=2,
+                     auto_prompt_reset=False)
+    except SSHLoginException:
+        pytest.fail("user failed to login")
```

Comment:
With this You can't expect/test failed login. The test will fail even if You want the user to be denied.